cmd/kube-scheduler/app/options/options.go

```diff
@@ -158,16 +149,14 @@ func (o *Options) Flags() (nfs apiserverflag.NamedFlagSets) {
 	return nfs
 }
 
-// ApplyTo applies the scheduler options to the given scheduler app configuration.
-func (o *Options) ApplyTo(c *schedulerappconfig.Config) error {
+// applyConfigs applies the scheduler options to the given scheduler app configuration.
+func (o *Options) applyConfigs() error {
```

Comment:
What do you think about putting this whole func inline in `Initialize`? It seems a bit weird to have this function called from exactly one place.

Reply:
Done